docs/src/content/docs/for-administrators/sequence-reporting.md

```diff
@@ -9,10 +9,10 @@ You can enable it in the `values.yaml` by configuring the `sequenceFlagging` sec
 
 ```yaml
 sequenceFlagging:
-  github:
-    organization: pathoplexus
-    repository: curation_reports
-    issueTemplate: sequence-metadata-issue.md  # (optional)
+    github:
+        organization: pathoplexus
+        repository: curation_reports
+        issueTemplate: sequence-metadata-issue.md # (optional)
```

Comment:
Not a fan of the 4 spaces indent for yaml - it's not very common to do this I think.

Reply:
I made an override for this.

Reply:
4 is default sometimes due to Python I think, but agree 2 is better